internal/build/resolver/current_userid.go

```diff
@@ -0,0 +1,24 @@
+package resolver
+
+import (
+	"context"
+
+	"github.com/buildkite/cli/v3/internal/build"
+	pipelineResolver "github.com/buildkite/cli/v3/internal/pipeline/resolver"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/go-buildkite/v3/buildkite"
+)
+
+func ResolveBuildFromCurrentUserId(uuid string, pipelineResolver pipelineResolver.PipelineResolverFn, f *factory.Factory) BuildResolverFn {
```

Comment:
This isn't necessarily the current users ID. It could be any user so we should remove that from the name